asmcli/lib/validate.sh

```diff
@@ -694,6 +712,59 @@ arg_required() {
   fi
 }
 
+validate_create_mesh_args() {
```

Comment:
If this is create_mesh specific I think we can leave it in the command file for now.

Reply:
Done